andel-vitorio/educators-report
# Development

## Description

This pull request adds a new feature to the application: the courses window and its associated forms for registration and editing. This functionality will allow users to manage information about courses more effectively.

## Changes

- Added a new "Disciplinas" window to the application.
- Implemented a course registration form.
- Implemented a course editing form.
- Provided fields for entering code, name, description, timetable, room, teacher, requirements, workload, credits and number of places.
- Customized the appearance of the window and forms to maintain consistency with the application's design.

## How to Test

1. Open the "Disciplinas" window in the application.
2. Use the registration form to add new courses.
3. Use the edit form to update existing course information.
4. Check that the information is saved correctly and reflects the changes made.
